common_benchmark_suite/openxla/benchmark/comparative_suite/jax/scripts/generate_model_artifacts.py

```diff
@@ -78,6 +79,16 @@ def _generate_artifacts(model: def_types.Model, save_dir: pathlib.Path,
 
     print(f"Completed generating artifacts {model.name}\n")
 
+    if auto_upload:
+      subprocess.run(
+          f"gsutil cp -r {model_dir} {GCS_UPLOAD_DIR}/{save_dir.name}/{model_dir.name}",
```

Comment:
This will have the problem of command injection. I'm wondering if the below would work?
```suggestion
        subprocess.run(["gcloud", "storage", "cp", "-r", model_dir, f"{GCS_UPLOAD_DIR}/{save_dir.name}/{model_dir.name}"], check=True)
```

Reply:
Changed.